### PR TITLE
Feature/summaries

### PR DESCRIPTION
**Pull request by trmdy**
This PR adds summaries, i.e. referater. There are a few things to note:

* We have hardcoded all types of summaries. This might be changed in the future if we need dynamic summary types (i.e. they change a lot).
* The "list" view will be expanded a lot given the current design spec.

### CODE REVIEW

File: summaries/consts.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
+SUMMARY_TYPE_OTHER = 'ot'
```
Comment (chdefrene) [inline summaries/consts.py]: Is there a particular reason for using two-letter combinations as keys? I remember we had a similar discussion when adding enums for KSG positions, where the conclusion was that we were to write it out in full text.

File: summaries/tests.py [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+        cls.users = [
+            User(username='user_one', email='user_one@example.com'),
+            User(username='user_two', email='user_two@example.com'),
+            User(username='user_three', email='user_three@example.com')
+        ]
+        for user in cls.users:
+            user.save()
+        cls.summary = Summary(
+            summary_type=SUMMARY_TYPE_BS,
+            contents="Kjersti falt av stolen.",
```
Comment (chdefrene) [inline summaries/tests.py:28]: LOL

**Review by chdefrene — APPROVED** [review]
OK, but please see the comments

File: summaries/consts.py (hunk 1 above)
Comment (trmdy) [reply summaries/consts.py]: No, not really. It is just what I typically use when I program 'choices', so it's likely my muscle memory at work here. I will change it to the more verbose versions that we agreed on.

File: summaries/tests.py (hunk 2 above)
Comment (trmdy) [reply summaries/tests.py:28]: :smile:

**Comment by trmdy** [comment]
The issues in the comments are fixed. Merging.

**Pull request merged**